docs/02-guides/extend-the-graphql-schema.mdx

```diff
@@ -54,10 +52,12 @@ to the extension containing a schema and some resolvers.
 For that, you can create the GraphQL module definition in the file
 `example-extensions/click-counter/graphql/index.ts` with the following content:
 
-```typescript title="example-extensions/click-counter/graphql/index.ts"
-export default {
+```typescript title="example-extensions/click-counter/graphql.ts"
```

Comment:
**issue:** I think we should document as we use it in the core. All modules are in an `index.ts` so that the module definition can be in the same dir than runtime and loaders

**suggestion:** restore `graphql/index.ts` that will be more consistent with the `runtime.ts` introduced below.

_same for other occurrences_